ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -542,7 +545,7 @@ impl AotIRDisplay for Instruction {
         }
 
         let mut ret = String::new();
-        if self.is_def(m) {
+        if let Some(_) = self.type_(m) {
```

Comment:
I don't understand the motivation here. It makes the code less readable, doesn't it?

Reply:
I guess it's not a big deal.